ci_framework/playbooks/01-bootstrap.yml

```diff
@@ -2,6 +2,11 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   tasks:
+    - name: Fail task execution when playbook is running as root.
```

Comment:
I'd rather get a reusable role for those kind of validations. We also have to ensure the reproducer.yml (and subsequent clean plays) aren't called with root. Having a proper role, allowing to extend it in an easy way without having to edit 4 places, would be far easier to maintain.